### DISCUSSION TITLE
Render dxf file on a window

### DISCUSSION THREAD

**Discussion by alireza2874** [mozman/ezdxf#1170] [Q&A]
Hi
Is there a way to preview dxf data in my UI inside python? I need to zoom and pan rendered dxf

**Comment by mbway** [comment]
not sure what you mean by 'my UI' but have you tried `ezdxf view <file>` or used the [drawing addon pyqt backend](https://ezdxf.readthedocs.io/en/stable/addons/drawing.html#pyqtbackend)?

**Reply by alireza2874** [reply]
I mean in my software user can open a window to preview the dxf that automatically created and after that save the file as dxf format.

**Reply by mbway** [reply]
and why isn't the pyqt (or matplotlib) drawing backend sufficient for that purpose?
